### ISSUE TITLE
Updating to juju/utils/v3?

### ISSUE THREAD

**Issue by gibmat** [canonical/candid#136]
Is there a plan to update to juju/utils/v3? Currently candid is on v2, and makes use of utils/debugstatus which was removed in v3. However, it would be nice if candid could update to v3 of juju/utils so it's not stuck on an older version of that library.

**Comment by gibmat** [comment]
Sorry, but juju/utils still seems to be pinned to version v2.0.0-20210305225158-eedbe7b6b3e2

**Comment by gaborbk** [comment]
Indeed, apologies @gibmat I forgot to include that change in my last PR. Will do so in the next one today.
